webrender/res/prim_shared.glsl

```diff
@@ -376,37 +376,45 @@ PrimitiveInfo fetch_text_run_glyph(int index, out vec4 color, out vec4 uv_rect)
 struct Image {
     PrimitiveInfo info;
     vec4 st_rect;               // Location of the image texture in the texture atlas.
-    vec4 stretch_size_uvkind;   // Size of the actual image.
+    vec2 stretch_size;          // Size of the actual image.
+    vec2 tile_spacing;          // Amount of space between tiled instances of this image.
+    float uvkind;
```

Comment:
This will need to be 2 vec4s. It's the only reliable way I've found of making all the shader layouts work correctly between linux and mac implementations.